src/crypto-providers/util.ts

```diff
@@ -121,6 +121,8 @@ const pathEquals = (path1: BIP32Path, path2: BIP32Path) => (
   path1.length === path2.length && path1.every((element, i) => element === path2[i])
 )
 
+const isByronPath = (path: BIP32Path) => classifyPath(path) === PathTypes.PATH_WALLET_SPENDING_KEY_BYRON
```

Comment:
I don't like this much, the name is deceptive --- the function returns false for a Byron account path. Do we need a "global" function like this?

Reply:
It's used on several places in two files, but the condition is simple, so maybe we could inline it.

Reply:
Inlined